src/routes/docs/tutorials/nuxt/step-6/+page.markdoc

```diff
@@ -0,0 +1,79 @@
+---
+layout: tutorial
+title: Add database
+description: Add databases and queries for ideas in your Nuxt project.
+step: 6
+---
+
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas.
+
+{% only_dark %}
+![Create project screen](/images/docs/tutorials/dark/idea-tracker-collection.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/tutorials/idea-tracker-collection.png)
+{% /only_light %}
+
+Create a new collection with the following attributes:
+| Field       | Type   | Required |
+|-------------|--------|----------|
+| userId      | String | Yes      |
+| title       | String | Yes      |
+| description | String | No       |
+
+Change the collection's permissions in the settings to give access.
+First, add an "Any" role and check the `READ` box.
+Next, add a "Users" role and give them access to create, update and delete.
+
+# Query methods {% #query-methods %}
+Now that you have a collection to hold ideas, we can use it to get, add and remove data from our app.
+We will create a new composable to manage the data fetching and their global state.
+Create a new file in the composables directory, `src/composables/useIdeas.js` and add the following code.
+
+```js
+import { ID, Query } from "appwrite";
+import { database } from "~/appwrite";
+import { ref } from "vue";
+
+const IDEAS_DATABASE_ID = "YOUR_DATABASE_ID";
```

Comment:
This should be lifted to a config file.

Reply:
Normally I'd have tucked this away in the`.env`-file, however that's not how you've constructed the other tutorials. 

This file is the equivalent of the `store` in SvelteKit
or `context`in React and follows their pattern.

Reply:
I would still use a `.env` file.

Reply:
I think the other tutorials should follow this pattern too :) 

Not conflicting. We're still improving existing tutorials, too :D

Reply:
`.env` file added 👯